### ISSUE TITLE
CLI Maintainer Nomination for Ihar Hrachyshka

### ISSUE THREAD

**Issue by RobotSail** [instructlab/instructlab#1953]
*For details on how this process works, please see our [contributor roles documentation](https://github.com/instructlab/community/blob/main/CONTRIBUTOR_ROLES.md)*

**Which member are you nominating for a team change?**
@booxter 

**Which team are you nominating for this member for?**
CLI Maintainers

**Make your case here: Why should the nominated member be on the proposed team?**

Ihar has made numerous contributions to the CLI repository in recent weeks, including both PRs and PR reviews.
Looking at the [community insights](https://github.com/instructlab/instructlab/pulse), Ihar is among the top contributors of the overall project. 

Ihar has provided us with [53 overall pull requests](https://github.com/instructlab/instructlab/pulls/booxter), [28 that have already been merged](https://github.com/instructlab/instructlab/pulls?q=is%3Apr+author%3Abooxter+is%3Amerged) and 8 that are currently open.

When it comes to pull requests, Ihar is often among the first to give a review to help it get through the pipeline, often providing valuable insight into what could be fixed and improved. Ihar invests a lot of time to ensure that all of the contributions being made are getting looked at adequately. 

In conclusion, Ihar helps the project be a vibrant place with the level of his contributions, and would make a great CLI maintainer.

**Comment by nathan-weinberg** [comment]
Just as a note - if approved @booxter should be moved _off_ the Triage team and added _on_ the Maintainer team - no one should be a member of both IMO (it doesn't make sense)

**Comment by alimaredia** [comment]
@RobotSail @nathan-weinberg What's the update on this? Is there a specific nomination period (ex: 5 or 7 days)?

**Comment by cdoern** [comment]
this has 8 thumbs up and no objections so I think we should do it

**Comment by nathan-weinberg** [comment]
@alimaredia the policy is whatever is in the already-linked doc: https://github.com/instructlab/community/blob/main/CONTRIBUTOR_ROLES.md

if something is not clear, welcome a PR to make any clarifications!

**Comment by cdoern** [comment]
> @alimaredia the policy is whatever is in the already-linked doc: https://github.com/instructlab/community/blob/main/CONTRIBUTOR_ROLES.md
> 
> if something is not clear, welcome a PR to make any clarifications!

https://github.com/instructlab/community/blob/main/governance.md#decision-making-at-the-instructlab-org-level this is the link under: `After a [decision has been made](https://github.com/instructlab/community/blob/main/governance.md#decision-making-at-the-instructlab-org-level), a Maintainer will create a PR to add you in the [MAINTAINER file](https://github.com/instructlab/community/blob/main/MAINTAINERS.md) within three weeks.` which results in a 404

**Comment by bjhargrave** [comment]
The PR will automatically be opened when the GitHub team is updated.

**Comment by RobotSail** [comment]
@bjhargrave I just moved @booxter from CLI Triagers to Maintainers, let's see what happens
